src/wp-includes/functions.php

```diff
@@ -7503,20 +7503,7 @@ function get_tag_regex( $tag ) {
  * @return bool Whether the slug represents the UTF-8 encoding.
```

Comment:
This should include an `@see _is_utf8_charset` since this is now essentially a wrapper function